server/index.ts

```diff
@@ -47,6 +48,9 @@ app
     // Load Settings
     const settings = getSettings().load();
 
+    // Load regions from tmdb
+    await new TheMovieDb().getRegions();
```

Comment:
`getRegions` is called once here so they will be stored for any later calls. Does this approach make sense for retrieving and storing them when the server starts up?